src/layouts/header/Header.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import { PROFILE_DATA } from "../../apis/api";
+import "./Header.css";
+
+const Header = () => {
+  const [profileData, setProfileData] = useState(null);
+
+  const GetProfileData = async () => {
+    const response = await PROFILE_DATA();
+    if (response === null) {
+      setProfileData(null);
+    } else {
+      const data = response.data;
+      setProfileData({ ...data });
+    }
+  };
+
+  useEffect(() => GetProfileData(), []);
```

Comment:
@youngjin-korea 

[수정요청]

에러로그가 발생하고 있어요. 수정 부탁드립니다.

```suggestion
  useEffect(() => {
  GetProfileData()}
  , []);
```

<img width="727" alt="image" src="https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/assets/56149367/b8e0feb1-1b3d-4e64-9b85-0aed6d71f897">


참고 부탁드립니다.

- https://react.dev/reference/react/useEffect#fetching-data-with-effects